HLTrigger/Configuration/python/customizeHLTforPatatrack.py

```diff
@@ -225,11 +224,10 @@ def customisePixelLocalReconstruction(process):
 # customisation for running the "Patatrack" pixel track reconstruction
 def customisePixelTrackReconstruction(process):
 
-    if not 'HLTRecoPixelTracksSequence' in process.__dict__:
+    if not all(seq in process.__dict__ for seq in ['HLTRecoPixelTracksSequence', 'HLTRecopixelvertexingSequence']):
```

Comment:
do we need/want to support the case where the configuration has `HLTRecoPixelTracksSequence` but not `HLTRecopixelvertexingSequence` ?

Reply:
I couldn't find a use case like this in the `GRun` menu, but supporting this seems easy enough.

Tentatively added in `e397a38176c`.

Checked that `e397a38176c` works as intended, by testing a config with
```diff
+process.MC_PixelTracksOnly_v0 = cms.Path(process.HLTRecoPixelTracksSequence)
+process.setSchedule_(cms.Schedule(process.MC_PixelTracksOnly_v0))
+process.prune()

#User-defined customization functions
from HLTrigger.Configuration.customizeHLTforPatatrack import customizeHLTforPatatrackTriplets
process = customizeHLTforPatatrackTriplets(process)
```

Reply:
Nice, thanks @missirol .